services/121-service/src/user/user.service.ts

```diff
@@ -183,7 +172,7 @@ export class UserService {
       {
         id: user.id,
         email: user.email,
-        role: user.role,
+        roles: user.roles,
```

Comment:
Here the JWT should only include the "text"-version of the roles, not their label/description...